packages/cita-signer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cryptape/cita-signer",
-  "version": "2.5.2",
+  "version": "2.5.3",
```

Comment:
😢 now only this modification committed.

But please have attention that the upgrade should be done with `@cryptape/cita-sdk` by the help of `lerna`

For example, the [dependency](https://github.com/cryptape/cita-sdk-js/blob/develop/packages/cita-sdk/package.json#L38) should be updated too, and `cryptape/cita-sdk` should be [upgraded](https://github.com/cryptape/cita-sdk-js/blob/develop/packages/cita-sdk/package.json#L3) too.

And the changelog should be updated meanwhile.

Please use 
```sh
$ lerna version minor --conventional-commits --exact --no-git-tag-version --no-push
```

to do that.

Reply:
And usually, this step should be done by the SDK publisher.

Reply:
> 😢 now only this modification committed.
> 
> But please have attention that the upgrade should be done with `@cryptape/cita-sdk` by the help of `lerna`
> 
> For example, the [dependency](https://github.com/cryptape/cita-sdk-js/blob/develop/packages/cita-sdk/package.json#L38) should be updated too, and `cryptape/cita-sdk` should be [upgraded](https://github.com/cryptape/cita-sdk-js/blob/develop/packages/cita-sdk/package.json#L3) too.
> 
> And the changelog should be updated meanwhile.
> 
> Please use
> 
> ```shell
> $ lerna version minor --conventional-commits --exact --no-git-tag-version --no-push
> ```
> 
> to do that.


>Why does it always report  "error get-caller-file@2.0.5"? We don't find 'get-caller-file@2.0.5' when we searched the packages-lock.json file.

Reply:
It looks like a bug from `npm`, I just rebuilt the [develop branch](https://github.com/cryptape/cita-sdk-js/pull/216) which has passed the CI before, and it failed this time. 

After then, I bumped the node to 10.16 in CI [PR](https://github.com/cryptape/cita-sdk-js/pull/250) and it passed.